docs/WritingTests.asciidoc

```diff
@@ -51,13 +51,15 @@ Here is an example in Perl:
 
 [source,perl]
 -------------------------------------------------------------------
-use Mojo::Base "basetest";
+use Mojo::Base "basetest"; # See note <1>
```

Comment:
Hm, keeping that in the code example means that people either copy the comment verbatim with the dangling reference or they need to remove that comment. How about we just keep the note below in line 62 and you don't change the code example itself?